src/unread/__tests__/unreadModel-test.js

```diff
@@ -231,15 +231,24 @@ describe('stream substate', () => {
 
   describe('EVENT_UPDATE_MESSAGE_FLAGS', () => {
     const mkAction = args => {
-      const { all = false, messages, flag = 'read', op = 'add' } = args;
+      const {
+        all = false,
+        allMessages = eg.makeMessagesState([]),
```

Comment:
Looks like this change doesn't get used.

This property (which is kind of a hack) only gets consumed in the `flags` reducer anyway, so it's natural that none of these tests would need to provide it. Best to let these tests forget its existence as much as possible, until the hack gets removed.